lib/timers/timer.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 # Released under the MIT License.
+# Copyright, 2025, by Patrik Wenger.
```

Comment:
This is great, but please at it at the bottom, the list is sorted by first contribution date. Thanks :)